### PR TITLE
Fix an issue in the python docs for KafkaTools.streamTableToAppendTable.

### PR DESCRIPTION
**Pull request by jcferretti**

### CODE REVIEW

File: Generators/src/main/java/io/deephaven/pythonPreambles/KafkaToolsPreamble.txt [hunk 1]
```diff
@@ -319,9 +319,9 @@ def simple(column_name:str, data_type:dh.DataType = None):
 @_passThrough
 def streamTableToAppendTable(t):
     """
-    Creates a 'stream' table from an 'append' type.
+    Creates an 'append' table from a 'stream' table type.
 
-    :param t:  The 'stream' table input.
-    :return:  The resulting 'append' table.
+    :param t:  The 'append' table input.
```
Comment (rcaudy) [inline Generators/src/main/java/io/deephaven/pythonPreambles/KafkaToolsPreamble.txt:324]: This wasn't wrong before, but you've made it so.

File: Integrations/python/deephaven/KafkaTools/__init__.py [hunk 2]
```diff
@@ -321,10 +321,10 @@ def simple(column_name:str, data_type:dh.DataType = None):
 @_passThrough
 def streamTableToAppendTable(t):
     """
-    Creates a 'stream' table from an 'append' type.
+    Creates an 'append' table from a 'stream' table type.
 
-    :param t:  The 'stream' table input.
-    :return:  The resulting 'append' table.
+    :param t:  The 'append' table input.
```
Comment (rcaudy) [inline Integrations/python/deephaven/KafkaTools/__init__.py:326]: This, too.

**Review by rcaudy — CHANGES_REQUESTED** [review]
You seem to have suffered under the Law of Conservation of Errors.

**Comment by jcferretti** [comment]
I got completely confused.  Abandoning this PR.

**Pull request closed without merging**